api/marketplace.py

```diff
@@ -44,7 +44,7 @@ def holding_cost_rate(self, merchant_id: str) -> float:
         response = self.request('get', 'holding_cost_rate/' + merchant_id)
         return response.json()
 
-    def register(self, endpoint_url_or_port: Union[str, int], merchant_name: str,
+    def register(self, endpoint_url_or_port: Union[str, int], merchant_name: str, color: str,
```

Comment:
What happens if color not set? Have you thought about a boring default such as '#gray'?

Reply:
If the pricers_merchant is used as a framework we will always have a defined color. Also if others use this function, they will know that register needs a color as input, so I don't see any reason for a default.

Reply:
Makes sense.